app/models/concerns/conclusion_reviews/pat_pdf.rb

```diff
@@ -497,4 +498,35 @@ def pat_receiver
 
       setting&.value || DEFAULT_SETTINGS[:conclusion_review_receiver][:value]
     end
+
+    def put_pat_annexes_on pdf
+      if annexes.any?
+        pdf.start_new_page
+
+        pdf.text Annex.model_name.human(count: 0).upcase, align: :center, style: :bold
+        pdf.move_down PDF_FONT_SIZE * 2
+
+        annexes.each do |annex|
+          pdf.text "#{Annex.human_attribute_name('title')}:", style: :bold
+          pdf.text annex.title
```

Comment:
```suggestion
          pdf.text annex.title, style: :bold
```
Creo que los pondría derecho, sin la etiqueta, veamos si así les parece, sino lo revertimos a esto.

Reply:
perfecto